apps/builder/app/builder/features/settings-panel/props-section/animation/animation-panel-content.tsx

```diff
@@ -332,16 +323,19 @@ const simplifiedRanges = [
     "cover 0%",
     <RangeCoverIcon />,
     ["cover", { type: "unit", unit: "%", value: 0 }],
+    "the subject just begins to appear in view",
```

Comment:
for consistency with others
```suggestion
    "when the subject just begins to appear in view",
```